modules/tow-boot/config.nix

```diff
@@ -69,7 +69,7 @@ in
       CMD_BDI = yes;
       CMD_CLS = yes;
       CMD_SETEXPR = yes;
-      CMD_PAUSE = yes;
+      CMD_PAUSE = lib.mkIf (!config.Tow-Boot.buildUBoot) yes;
```

Comment:
`CMD_PAUSE` has been mainlined, so it will be possible to re-enable this when upgrading U-Boot.